index.html

```diff
@@ -217,6 +217,20 @@ <h2 class="fz20px">Skills</h2>
     </article>
   </main>
   <footer>
+    <article class="bg-footer border-tl">
+      <section class="footer_class p5vh">
+        <h2 class="title">Contact me</h2>
+        <p class="details">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+        <form class="flex flex-column" action="https://formspree.io/f/mlevqnqq" method="post">
```

Comment:
- [ ] Good work implementing a formspree to receive emails about queries. But for naming conventions kindly change the method to a uppercase, like `POST` 🥇